doc/requirements.txt

```
@@ -0,0 +1,4 @@
+IPython
+jupyter_client
+sphinx>=4.0.0
```

Let’s set a higher major version here if possible. Latest is 7.
```suggestion
sphinx>=7.0.0
```